### PR TITLE
Update co-occurence plots

### PR DESCRIPTION
**Pull request by jashapiro**
#### What scientific question is your analysis addressing?

Updating interaction plots to use consensus data.

#### What was your approach?

Change file name and rerun.

#### Which areas should receive a particularly close look?

The consensus data, since it has fewer calls overall, also has fewer high frequency calls with potentially significant co-occurences. The net result is that many of these plots have gotten much smaller, with fewer interactions plotted, but the largest of them (All disease) now has overall fewere high-frequency mutations, but more of them are "significant". I *think* this is a statistical artifact, but I feel like it bears more investigation. 

#### Is the analysis in a mature enough form that the resulting figure(s) and/or table(s) are ready for review?

Yes

#### What types of results are included (e.g., table, figure)?

Plots and tables of co-occurence.

#### Reproducibility Checklist

- [x] The dependencies required to run the code in this pull request have been added to the project Dockerfile.
- [x] This analysis has been added to continuous integration.
- [x] This analysis is recorded in the table in `analyses/README.md`.

### CODE REVIEW

File: analyses/interaction-plots/01-create-interaction-plots.sh [hunk 1]
```diff
@@ -21,8 +21,8 @@ ALL=${OPENPBTA_ALL:-1}
 ind_samples=data/independent-specimens.wgs.primary-plus.tsv
 metadata=data/pbta-histologies.tsv
 
-# using lancet data for now
-maf=data/pbta-snv-lancet.vep.maf.gz
+# using consensus
+maf=data/pbta-snv-consensus-mutation.maf.tsv.gz
 
 cooccur=${results_dir}/lancet_top50
```
Comment (jaclyn-taroni) [inline analyses/interaction-plots/01-create-interaction-plots.sh]:
```suggestion
cooccur=${results_dir}/consensus_top50
```

File: analyses/interaction-plots/01-create-interaction-plots.sh [hunk 2]
```diff
@@ -21,8 +21,8 @@ ALL=${OPENPBTA_ALL:-1}
...
 plot=${plot_dir}/lancet_top50
```
Comment (jaclyn-taroni) [inline analyses/interaction-plots/01-create-interaction-plots.sh]:
```suggestion
plot=${plot_dir}/consensus_top50
```

File: analyses/README.md [hunk 3]
```diff
@@ -15,11 +15,11 @@ Note that _nearly all_ modules use the harmonized clinical data file (`pbta-hist
...
 | [`copy_number_consensus_call`](https://github.com/jaclyn-taroni/OpenPBTA-analysis/tree/analyses-readme/analyses/copy_number_consensus_call) | `pbta-cnv-cnvkit.seg.gz`, `pbta-cnv-controlfreec.tsv.gz`, `pbta-sv-manta.tsv.gz` | *In progress*; will produce consensus copy number calls per [#128](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/128) | N/A
-| [`create-subset-files`](https://github.com/jaclyn-taroni/OpenPBTA-analysis/tree/analyses-readme/analyses/create-subset-files) | All files | This module contains the code to create the subset files used in continuous integration | All subset files for continuous integration
+| [`create-s*ubset-files`](https://github.com/jaclyn-taroni/OpenPBTA-analysis/tree/analyses-readme/analyses/create-subset-files) | All files | This module contains the code to create the subset files used in continuous integration | All subset files for continuous integration
```
Comment (jaclyn-taroni) [inline analyses/README.md]: This was probably an accidental update
```suggestion
| [`create-subset-files`](https://github.com/jaclyn-taroni/OpenPBTA-analysis/tree/analyses-readme/analyses/create-subset-files) | All files | This module contains the code to create the subset files used in continuous integration | All subset files for continuous integration
```

**Comment by jashapiro** [comment]
> The file names need to be updated to reflect the fact that you're using the consensus files now instead of Lancet (and the old Lancet files removed). 

Oops. Done.

**Pull request merged**